build.gradle

```diff
@@ -223,6 +223,8 @@ dependencies {
     compile group: 'io.circe', name: "circe-parser_${scalaBinaryVersion}", version: circeBinaryVersion
 
     compile group: 'com.typesafe.play', name: "play-json_${scalaBinaryVersion}", version: '2.6.3'
+    compile group: 'com.typesafe.play', name: 'play-ahc-ws-standalone_2.12', version: '2.0.3'
```

Comment:
Don't we already have an external http library?

Reply:
Yah, we actually have 2 - Akka HTTP and Apache Httpcomponents Fluent-HC. We probably shouldn't add a third...and in fact should reduce to one.

Reply:
this task was supposed to be implemented urgently. so... there was no time to think about this